src/FtpExtractor.php

```diff
@@ -20,6 +23,8 @@ class FtpExtractor
     private const FILE_TIMESTAMP_KEY = 'timestamp';
     private const FILE_SOURCE_KEY = 'source-path';
     private const LOGGER_INFO_LOOP = '10';
+    private const RETRY_CONNECTION = 3;
```

Comment:
"retry connection" mi přijde jako "mám se zkoušet připojit znova" (true/false), takže bych se přimlouval za "RETRY_CONNECTION_RETRIES" nebo jen "CONNECTION_RETRIES".

Reply:
Nebo třeba number of retries.

Reply:
Přejmenoval jsem na `CONNECTION_RETRIES`